hypertools/plot/plot.py

```diff
@@ -60,9 +59,9 @@ def plot(x, fmt=None, marker=None, markers=None, linestyle=None,
         A list of labels for each point. Must be dimensionality of data (x).
         If no label is wanted for a particular point, input None.
 
-    legend : list or bool
```

Comment:
I think `git rebase` would work if I hadn't made those commits. What it seems to be doing is it updates my master branch with the upstream (your master branch), and then it throws my commits on top of that. So these accidental changes are still there. There's probably a way to fix this in `git` but I'm not sure :confused:. Maybe I can just add the proper changes back into my local branch and commit that.